README.md

```diff
@@ -1,6 +1,6 @@
 <!-- markdownlint-disable MD033 -->
 <h1>
-    <img src="https://github.com/sighupio/fury-distribution/blob/main/docs/assets/fury-epta-white.png?raw=true" align="left" width="90" style="margin-right: 15px"/>
+    <img src="https://raw.githubusercontent.com/sighupio/fury-distribution/refs/heads/feat/rebranding/docs/assets/white-logo.png" align="left" width="90" style="margin-right: 15px"/>
```

Comment:
Let's not point to a branch. The branch will be deleted as soon as it gets merged into main, effectively breaking the image.

We can leave this as-is and when we merge the rebranding we will update all the readmes.